src/cli.ts

```diff
@@ -51,12 +96,12 @@ const getConfig = (env: any, args: string[]): SimulatorConfig =>
       'automatically associate device to account',
       false,
     )
+    .option('-v, --verbose', 'output debug info', false)
     .option(
       '-t, --app-type <appType>',
-      'Type of device application. "mss" or "atv2"',
-      'atv2',
+      'Specifies the shadow to use. For custom shadow, pass a JSON-encoded shadow object. Otherwise, pass "mss" or "atv2" to automatically generate a conformal shadow',
+      handleAppType,
```

Comment:
How hard would it be to add the ability to pass a filename containing the JSON? Could be big.

Reply:
It would just add another layer around it. Want me to try and add it to this ticket?

Reply:
Yes please. I don't think we should require all the JSON to be on the command line.